src/main/java/zd/zero/waifu/motivator/plugin/assets/RemoteAssetManager.kt

```diff
@@ -43,22 +43,22 @@ abstract class RemoteAssetManager<T : AssetDefinition, U : Asset>(
 
     private fun initializeAssetCaches(assetFileUrl: Optional<String>, breakOnFailure: Boolean = true) {
         assetFileUrl
-                .flatMap { assetUrl -> initializeRemoteAssets(assetUrl) }
-                .doOrElse({ allAssetDefinitions ->
-                    status = Status.OK
-                    remoteAndLocalAssets = allAssetDefinitions
-                    localAssets = allAssetDefinitions.filter { asset ->
-                        AssetManager.constructLocalAssetPath(assetCategory, asset.path)
-                            .filter { Files.exists(it) }
-                            .isPresent
-                    }.toSet().toMutableSet()
-                }) {
-                    if (breakOnFailure) {
-                        status = Status.BROKEN
-                        remoteAndLocalAssets = listOf()
-                        localAssets = mutableSetOf()
-                    }
+            .flatMap { assetUrl -> initializeRemoteAssets(assetUrl) }
+            .doOrElse({ allAssetDefinitions ->
+                status = Status.OK
+                remoteAndLocalAssets = allAssetDefinitions
+                localAssets = allAssetDefinitions.filter { asset ->
+                    AssetManager.constructLocalAssetPath(assetCategory, asset.path)
+                        .filter { Files.exists(it) }
+                        .isPresent
+                }.toSet().toMutableSet()
+            }) {
+                if (breakOnFailure) {
+                    status = Status.BROKEN
+                    remoteAndLocalAssets = listOf()
+                    localAssets = mutableSetOf()
                 }
+            }
     }
 
     fun supplyAssetDefinitions(): List<T> =
```

Comment:
This is only ever used in tests. Please remove.

Reply:
So, I removed it but reverted it now as PR #215 would use it.